spring-integration-zookeeper/src/test/java/org/springframework/integration/zookeeper/metadata/ZookeeperMetadataStoreTests.java

```diff
@@ -16,17 +16,6 @@
 
 package org.springframework.integration.zookeeper.metadata;
 
-import static org.hamcrest.Matchers.containsString;
```

Comment:
This imports reorder change is wrong from this project Checkstyle perspective.

Reply:
This passed checkstyle for me